collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java

```diff
@@ -296,45 +294,59 @@ protected void onResume() {
         boolean edit = sharedPreferences.getBoolean(
                 AdminKeys.KEY_EDIT_SAVED, true);
         if (!edit) {
-            mReviewDataButton.setVisibility(View.GONE);
-            mReviewSpacer.setVisibility(View.GONE);
+            if (mReviewDataButton!=null)
+                mReviewDataButton.setVisibility(View.GONE);
+            if (mReviewSpacer!=null)
+                mReviewSpacer.setVisibility(View.GONE);
         } else {
-            mReviewDataButton.setVisibility(View.VISIBLE);
-            mReviewSpacer.setVisibility(View.VISIBLE);
+            if (mReviewDataButton!=null)
```

Comment:
The code I think have curly braces all the time even though the if only contains 1 statement.